resources/views/admin/user/list.blade.php

```diff
@@ -0,0 +1,63 @@
+<?php ?>
+@extends('admin.layout.index')
+
+@section('content')
+    <div id="page-wrapper">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header">User</h1>
+            </div>
+            <!-- /.col-lg-12 -->
+        </div>
+        <!-- /.row -->
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        Manager User
+                    </div>
+                    <!-- /.panel-heading -->
+                    <div class="panel-body">
+                        <div class="table-responsive">
+                            <table class="table table-bordered table-hover">
+                                <thead>
+                                    <tr>
+                                        <th>ID</th>
+                                        <th>User</th>
+                                        <th>Email</th>
+                                        <th>Password</th>
+                                        <th></th>
+                                        <th></th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    <tr class="odd gradeX">
+                                        <td>1</td>
+                                        <td>Phan Quoc Tuan</td>
```

Comment:
Why are you using mock data in here?

Reply:
This is just an interface initialization, this is just fake data of the user management screen, not to get the database

Reply:
No need using mock data

Reply:
I fixed, no need mock data, only layout